script.js

```diff
@@ -0,0 +1,156 @@
+//현재 날짜와 시간을 가져오기
+const currentDate = new Date();
+const year = currentDate.getFullYear();
+const month = currentDate.getMonth() + 1;
+const day = currentDate.getDate();
+//요일 구하기
+function getWeekDay() {
+  const week = ["일", "월", "화", "수", "목", "금", "토"];
+  const weekDay = week[currentDate.getDay()];
+  return weekDay;
+}
+const weekDay = getWeekDay(); //요일 함수 선언
+const yyyy_mm_dd = `${year}년 ${month}월 ${day}일 ${weekDay}요일`;
+//오늘의 날짜 띄우기
+document.getElementById("todayDate").textContent = yyyy_mm_dd;
+
+//페이지 로드 시에 로컬스토리지에서 가져오기
+document.addEventListener("DOMContentLoaded", loadList);
+
+//버튼 클릭 시, 할일 저장
+const submitBtn = document.getElementById("submitBtn");
+submitBtn.addEventListener("click", (event) => {
+  event.preventDefault(); //
+  const input = document.querySelector("input");
+  const text = input.value.trim(); //양 끝 공백 제거 후 text에 저장
```

Comment:
이번 과제에서는 input이 하나여도 상관없지만, 이왕 사용자로부터 입력받는 input에 id를 지정했으니까 id로 찾는건 어떨까요??
그리고 변수명도 한 눈에 봤을 때 어떠한 값을 갖고 있는 지 분명했으면 좋겠어요.
```suggestion
  const inputEl = document.querySelector("#todoRecord");
  const userInput = inputEl.value.trim(); //양 끝 공백 제거 후 text에 저장
```